pkg/controller/deployment/deployment_controller_test.go

```diff
@@ -362,7 +362,7 @@ func TestReentrantRollback(t *testing.T) {
 }
 
 // TestPodDeletionEnqueuesRecreateDeployment ensures that the deletion of a pod
-// will requeue a Recreate deployment iff there is no other pod returned from the
+// will requeue a recreate deployment if there is no other pod returned from the
```

Comment:
iff = if and only if, that's a valid statement, not a typo.

Reply:
TIL